### PR TITLE
i#1960 add new drltace tests: arguments and ret value checks

### PR DESCRIPTION
**Pull request by mxmssh**
Added new test for library calls, arguments and ret addresses printing
Added new test for symargs printing
Fixed a bug when drltrace was not able to print in the console

### CODE REVIEW

**Comment by mxmssh** [comment]
May be I can assign for somebody else (just asking for future pull requests) to keep work balance ?

**Comment by mxmssh** [comment]
Hm, are these failures related to drstrace_unit_tests failure ? How can I check that ?

**Comment by mxmssh** [comment]
ok, looked into the wrong log. We have one flaky test failure and pcache-use.

File: drltrace/CMakeLists.txt [hunk 1]
```diff
@@ -117,10 +117,33 @@ copy_and_adjust_drpaths(${CMAKE_RUNTIME_OUTPUT_DIRECTORY} drltrace)
...
+
+  #regex strings for libcalls and arguments printing test
+  set(libcall_args1_01 "    arg 0: 0x([0-9a-f]+)\n    arg 1: 0x([0-9a-f]+)\n")
+  set(libcall_ret "    and return to module id:([0-9]+), offset:0x([0-9a-f]+)\n")
+if (WIN32)
+  set(libcall_args2_0 "    arg 0: 0x([0-9a-f]+) \\(type=HANDLE, size=0x([0-9a-f]+)\\)\n")
+  set(libcall_args2_1 "    arg 1: 0x([0-9a-f]+) \\(type=int, size=0x([0-9a-f]+)\\)\n")
+
+  set(libcall_name1 "~~([0-9a-f]+)~~ KERNELBASE.dll!VirtualQuery\n")
+  set(libcall_name2 "~~([0-9a-f]+)~~ ntdll.dll!ZwQueryInformationProcess\n")
```
Comment (derekbruening) [inline drltrace/CMakeLists.txt]: So we test that drltrace is printing out one syscall's 1st 2 args and that they are of a certain type.  OK, seems a reasonable starting point.

**Comment by mxmssh** [comment]
Automatic merging is blocked, could you merge manually ?

**Comment by derekbruening** [comment]
In general, please list the issue number of failing tests here so we know they are in the tracker.  This pcache-use failure is the premature deletion assert listed in #1938.

**Pull request merged**